src/api/comments.js

```diff
@@ -0,0 +1,24 @@
+import { request, remove, post } from './api';
+
+const query = '/comments';
+
+export function getComments(postId) {
+  return request(query)
+    .then((result) => {
+      const postComments = result.filter(comment => (
+        comment.postId === postId
+      ));
```

Comment:
you you should fetch only comments of a particular post, not filter them on the client
use url parameter for this